prover/crypto/keccak/subtle/xor_amd64.s

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 The Go Authors. All rights reserved.
```

Comment:
Likewise, we should remove the header or avoid these optimization if we copy non-apache code.